src/firmware/app/control/python_controller/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy==1.18.4
+scipy==1.4.1
+control==0.8.3
+matplotlib==3.2.1
+pyparsing==2.4.7
+cycler==0.10.0
+kiwisolver==1.2.0
```

Comment:
Are these being used? Are you leaving these here because we'll use them in the future?

Reply:
They are actually currently used

Reply:
can you point me to where?

Reply:
Some of them are hidden inside of other packages.

For example, the python control toolbox uses kiwi solver.

If I don't include kiwi solver here the bazel can't build it

Reply:
But I was able to remove the requirements from the _test.py file 👍

Reply:
Oh I see